xrfeitoria/data_structure/models.py

```diff
@@ -21,6 +21,46 @@
 class RenderPass(BaseModel):
     """Render pass model contains render layer and image format.
 
+    Supported render layer and image format:
+
+        .. tabs::
+            .. tab:: RenderLayerBlender
```

Comment:
Just to confirm that it is `RenderLayer` rather than `RenderOutput` here?

Reply:
Yes, it is `RenderLayer` here. `RenderOutput` is a concept containing more info like camera_params.